cmd/completion.go

```diff
@@ -25,17 +25,17 @@ func newCompletionCmd(rootCmd *cobra.Command) *cobra.Command {
 	// bash completions
 	var bashCompletionCmd = &cobra.Command{
 		Use:   "completion",
-		Short: "Generates bash tab completions",
-		Long: `Outputs a bash completion script for appsody to stdout.  Bash completion is optionally available for your convenience. It helps you fill out appsody commands when you type the [TAB] key.
+		Short: "Generate bash tab completions.",
+		Long: `Generate a bash completion script for Appsody to stdout.  Bash completion is optionally available for your convenience. It helps you fill out Appsody commands when you type the [TAB] key.
```

Comment:
Do we need to mention stdout here? Might be a bit clearer if we just talk about the auto complete.